bin/aursync

```diff
@@ -33,6 +37,13 @@ clone() {
                 warning "remote $remote does not match $repo"
             fi
 
+            # Update the config to get the newest pubkey. Is this smart or not efficient enough?
```

Comment:
I thought about this while writing the other stuff.

`time` tells me the performance impact is negligible, but that is on a fast machine and you or others might disagree.